src/systemverilog/systemverilog.ts

```diff
@@ -319,9 +319,7 @@ export const language = <ILanguage>{
 	// The main tokenizer for our languages
 	tokenizer: {
 		root: [
-			// C++ 11 Raw String
-			[/@encoding?R\"(?:([^ ()\\\t]*))\(/, { token: 'string.raw.begin', next: '@raw.$1' }],
```

Comment:
since you got started getting rid of this can you also go ahead and finish it and remove all the references to the raw string in this file? Thanks! (yes im asking you to do my section for me :P)